MIGRATING.md

```diff
@@ -0,0 +1,17 @@
+## Migrating to Essential Next.js Plugin
+
+This guide is to assist `next-on-netlify` users in their migration to Netlify's [Essential Next.js build plugin](https://github.com/netlify/netlify-plugin-nextjs).
```

Comment:
[sand] `Build Plugin` is capitalized, per style guide. `plugin` on its own (no build), is lowercase.
```suggestion
This guide is to assist `next-on-netlify` users in their migration to Netlify's [Essential Next.js Build Plugin](https://github.com/netlify/netlify-plugin-nextjs).
```